lib/elixir_analyzer/exercise_test/assert_call/compiler.ex

```diff
@@ -69,7 +70,9 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall.Compiler do
   Handle the final result from the assert function
   """
   @spec handle_traverse_result({any, map()}) :: boolean
-  def handle_traverse_result({_, %{found_called: found}}), do: found
+  def handle_traverse_result({_, %{found_called: found, calling_fn: calling_fn} = acc}) do
+    found || (!is_nil(calling_fn) && indirect_call?(acc))
```

Comment:
What's the difference between the situation when `found` is truthy vs when `called_fn in tree[calling_fn]` is truthy in `indirect_call?`?